src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe.sarif

```diff
@@ -0,0 +1,887 @@
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2018",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass_NoSEH",
+            "arguments": [
+              "Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2019",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass",
+            "arguments": [
+              "Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2021",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass",
+            "arguments": [
+              "Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2022",
+          "level": "notApplicable",
+          "formattedRuleMessage": {
+            "formatId": "NotApplicable_InvalidMetadata",
+            "arguments": [
+              "Managed_AnyCPU_VS2017_NoPrefer32Bit.exe",
+              "SignSecurely",
+              "image is not signed "
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+              }
+            }
+          ]
+        }
+      ],
+      "toolNotifications": [
+        {
+          "id": "MSG001.AnalyzingTarget",
+          "physicalLocation": {
+            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Managed_AnyCPU_VS2017_NoPrefer32Bit.exe"
+          },
+          "message": "Analyzing 'Managed_AnyCPU_VS2017_NoPrefer32Bit.exe'...",
+          "level": "note",
+        }
+      ],
+      "rules": {
+        "BA3001": {
+          "id": "BA3001",
+          "name": "EnablePIEOnExecutables",
+          "shortDescription": "A Position Independent Executable (PIE) relocates all of its sections at load time, including the code section, if ASLR is enabled in the Linux kernel (instead of just the stack/heap).",
```

Comment:
>PIE [](start = 66, length = 3)

Can we change casing of this to Pie? And ELF to Elf in any rule ids/names before we ship